rdflib/graph.py

```diff
@@ -379,6 +403,7 @@ def close(self, commit_pending_transaction=False):
         database or releasing some resource.
         """
         self.__store.close(commit_pending_transaction=commit_pending_transaction)
+        return self
```

Comment:
My only comment is that graph.close() probably shouldn't return self, because it doesn't make sense to chain any other methods after close().

Reply:
I changed it to an explicit return of the last command. Thanks for spotting :-)